keras/utils/conv_utils.py

```diff
@@ -58,20 +58,20 @@ def normalize_tuple(value, n, name):
     n: The size of the tuple to be returned.
     name: The name of the argument being validated, e.g. "strides" or
       "kernel_size". This is only used to format error messages.
-
+    allow_zero: Whether the function will accept zero values.
```

Comment:
Default to False. A ValueError will raised if zero is received and this param is False.

Reply:
Done